changelog/fix-7771-remove-instant-deposit-transaction-list

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Replace the deposits overview transactions list with a "transaction history is unavailable for instant deposits" message.
```

Comment:
`nit` Since we are changing the 'deposit details page', I think it should be singular.

```suggestion
Replace the deposit overview transactions list with a "transaction history is unavailable for instant deposits" message.
```